spring-cloud-gcp-data-datastore/src/test/java/com/google/cloud/spring/data/datastore/repository/query/GqlDatastoreQueryTests.java

```diff
@@ -447,6 +448,39 @@ public void pageableTestPageCursor() {
 				.queryKeysOrEntities(any(), eq(Trade.class));
 	}
 
+	@Test
+	public void streamResultTest() {
+		Mockito.<Class>when(this.queryMethod.getReturnedObjectType())
+				.thenReturn(Trade.class);
+		Parameters parameters = mock(Parameters.class);
+
+		when(this.queryMethod.getParameters())
+				.thenReturn(parameters);
+		when(parameters.getNumberOfParameters())
+				.thenReturn(0);
+		when(this.queryMethod.isStreamQuery()).thenReturn(true);
```

Comment:
nit: some of these have `.thenReturn` on a new line, some on the same line. I don't have a personal preference, as long as they all do the same thing within at least a method (well, unless a line is too long and one line does not fit, but in this case all statements are short).

Reply:
Got it. Making them all in one line since they are short in this case.